test/pde_systems/MOL_1D_Linear_Diffusion.jl

```diff
@@ -383,7 +383,7 @@ end
 end
 
 
-@testset "Test 06: Dt(u(t,x)) ~ Dxx(u(t,x)), time-dependent Robin BCs, Order 6" begin
+@test_broken begin #@testset "Test 06: Dt(u(t,x)) ~ Dxx(u(t,x)), time-dependent Robin BCs, Order 6" begin
```

Comment:
this breaks a test but doesn't fix one?

Reply:
That was the test that was already broken, check the coverage for the last merge. We didn't have a test for this case, I can add an MWE for what this fixes?

Reply:
yes add the MWE